tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
-minversion = 2.4.0
+minversion = 4.2.0
 envlist = py{39,310,311,312}
-isolated_build = True
```

Comment:
Why did you remove the isolated build? (I don't really use tox personally and I run the tests from CLI using specific Python version)

Reply:
All the context is in the commit messages. From [`f2f2e79` (#11602)](https://github.com/sphinx-doc/sphinx/pull/11602/commits/f2f2e79a9d5a221c50f3eeb08dc0dfce48ac5d08):

> `isolated_build` is no longer required since builds are isolated by default in tox 4.
> 
> `basepython` is no longer required since tox 4 only supports Python 3 and 'basepython' defaults to the version of Python that tox is installed with.

Reply:
Oh sorry, I didn't see the commit message itself. My bad !